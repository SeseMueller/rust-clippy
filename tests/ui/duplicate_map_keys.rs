#![warn(clippy::duplicate_map_keys)]
use std::collections::HashMap;

fn main() {
    let example = HashMap::from([(5, 1), (5, 2)]);
}
