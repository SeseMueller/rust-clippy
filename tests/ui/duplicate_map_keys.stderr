error: this `HashMap` has a hash collision and will lose data
  --> tests/ui/duplicate_map_keys.rs:5:19
   |
LL |     let example = HashMap::from([(5, 1), (5, 2)]);
   |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: consider using a different keys for all items
   = note: `-D clippy::duplicate-map-keys` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(clippy::duplicate_map_keys)]`

error: aborting due to 1 previous error

